src/core/Akka.Tests/Actor/ActorLifeCycleSpec.cs

```diff
@@ -281,24 +281,33 @@ public SupervisorTestActor(IActorRef testActor)
 
             protected override void OnReceive(object message)
             {
-                PatternMatch.Match(message)
-                    .With<Spawn>(m =>
-                    {
-                        Context.ActorOf(Props.Create(() => new KillableActor(testActor)), m.Name);
-                        testActor.Tell(("Created", m.Name));
-                    })
-                    .With<ContextStop>(m =>
-                    {
-                        var child = Context.Child(m.Name);
-                        Context.Stop(child);                    
-                    })
-                    .With<Stop>(m =>
-                    {
-                        var child = Context.Child(m.Name);
-                        ((IInternalActorRef)child).Stop();
-                    })
-                    .With<Count>(m => 
-                        testActor.Tell(Context.GetChildren().Count()));
+                switch (message)
+                {
+                    case Spawn m:
+                        {
+                            Context.ActorOf(Props.Create(() => new KillableActor(testActor)), m.Name);
+                            testActor.Tell(("Created", m.Name));
+                        }
+                        break;
+                    case ContextStop m:
+                        {
+                            var child = Context.Child(m.Name);
+                            Context.Stop(child);
+                        }
+                        break;
+                    case Stop m:
+                        {
+                            var child = Context.Child(m.Name);
+                            ((IInternalActorRef)child).Stop();
+                        }
+                        break;
+                    case Count m:
+                    default:
```

Comment:
This is wrong, there is no default handle.

Reply:
I should remove the default?